lib/ibm/cloud/sdk.rb

```diff
@@ -1,13 +1,43 @@
-require "json"
-require "rest-client"
+# typed: true
+# frozen_string_literal: true
 
-require "ibm/cloud/sdk/base_service"
-require "ibm/cloud/sdk/iam"
-require "ibm/cloud/sdk/power_iaas"
-require "ibm/cloud/sdk/resource_controller"
-require "ibm/cloud/sdk/version"
+require_relative 'sdk/base_service'
+require_relative 'sdk/iam'
+require_relative 'sdk/power_iaas'
+require_relative 'sdk/resource_controller'
+require_relative 'sdk/vpc'
 
 module IBM
+  # Holds the SDK pieces.
+  class CloudSDK
```

Comment:
This should match the file name so `IBM::Cloud::SDK`

Reply:
This is a work in progress. I just got it working properly yesterday, and am still evaluating how it functions in practice. I'll clean it up and do it properly when I am satisfied this is the correct solution.